syntax/tmux.vim

```diff
@@ -205,8 +205,8 @@ syn match  tmuxAttrSeparator /,/     contained display
 " Shell command interpolation
 syn region tmuxShellInpol matchgroup=tmuxShellInpolDelimiter start=/#(/ skip=/#(.\{-})/ end=/)/ contained keepend
 
-syn region tmuxString matchgroup=tmuxStringDelimiter start=/"/ skip=/\\./ end=/"/ contains=tmuxFmtInpol,tmuxFmtAlias,tmuxAttrInpol,tmuxShellInpol,tmuxPromptInpol,tmuxDateInpol,@Spell display keepend
-syn region tmuxString matchgroup=tmuxStringDelimiter start=/'/ end=/'/            contains=tmuxFmtInpol,tmuxFmtAlias,tmuxAttrInpol,tmuxShellInpol,tmuxPromptInpol,tmuxDateInpol,@Spell display keepend
+syn region tmuxString matchgroup=tmuxStringDelimiter start=/\([^\\]"\)\|\(^"\)/ skip=/\\./ end=/"/ contains=tmuxFmtInpol,tmuxFmtAlias,tmuxAttrInpol,tmuxShellInpol,tmuxPromptInpol,tmuxDateInpol,@Spell display keepend
+syn region tmuxString matchgroup=tmuxStringDelimiter start=/\([^\\]'\)\|\(^'\)/            end=/'/ contains=tmuxFmtInpol,tmuxFmtAlias,tmuxAttrInpol,tmuxShellInpol,tmuxPromptInpol,tmuxDateInpol,@Spell display keepend
```

Comment:
I think this change will make `\"foo"` a valid string, even though it's not.

Also, semantically the "key" in `bind-key` command is not a string: you can't do `bind-key "lorem ipsum" ...`

I propose we treat `\"` and `\'` as another version of `tmuxKey` group. So maybe add another line below line 163 that would match just these two keys.